packages/cli/fragments/fragment-clojurescript/deps.edn

```diff
@@ -0,0 +1,9 @@
+{:paths ["src/main/clojure"]
+
+ :deps {reagent/reagent {:mvn/version "1.1.1"
+                         :exclusions [cljsjs/react
+                                      cljsjs/react-dom
+                                      cljsjs/react-dom-server]}}
```

Comment:
this package seems to be deprecated https://clojars.org/cljsjs/react-dom-server

Reply:
This record means that we want to exclude some dependencies and use `React` directly from `npm`. But I double-checked and this record unnecessary. I've removed this part.